Packs/CommonScripts/Scripts/HttpV2/HttpV2.yml

```diff
@@ -0,0 +1,179 @@
+args:
+- default: false
+  description: Specify where the request should be sent. Include the URI scheme ('http'
+    or 'https').
+  isArray: false
+  name: url
+  required: true
+  secret: false
+- auto: PREDEFINED
+  default: false
+  description: Specify the HTTP method to use.
+  isArray: false
+  name: method
+  predefined:
+  - GET
+  - POST
+  - PUT
+  - PATCH
+  - DELETE
+  - HEAD
+  required: true
+  secret: false
+- default: false
+  description: Specify a hash of headers to send with the request.
+  type: keyValue
+  isArray: false
+  name: headers
+  required: false
+  secret: false
+- default: false
+  description: Specify the body of the request.
+  type: textArea
+  isArray: false
+  name: body
+  required: false
+  secret: false
+- default: false
+  description: |-
+    Specify the content type to use with the request. For example: application/json.
+    Mapped types are:
+    json  (application/json)
+    xml (text/xml)
+    form (application/x-www-form-urlencoded)
+    data (multipart/form-data)
+  isArray: false
+  name: request_content_type
+  required: false
+  secret: false
+- default: false
+  description: |-
```

Comment:
Please change to:

Specify the Accept header for the request - the response content type.
Shorthands are provided for the following common content types:
json (application/json)
xml (text/xml)
form (application/x-www-form-urlencoded)
data (multipart/form-data)
If you choose to define a different type, please include the full type name, e.g: application/pdf